### PR TITLE
Gherkin and automation of topology usability improvements

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-4944
Story: https://issues.redhat.com/browse/ODC-6500
Acceptance criteria:

-  SBR
    - Provide a visual indicator so that users know if there is an error with a SBR in topology
    - In Topology side panel, include status
    - In Details of Topology side panel, show the error
    - Show a status badge on the SB details page
    - Show a Status field in the right column of the SB details page
-  Remove the following features from topology view
    - Consumption mode can be removed, thus MODE in general is not needed
    - Layout 2 option, thus no layout options are needed
-  Topology should render differently based on zoom level ( actual percentages to be defined during development )
    - When zoomed to 50% scale, all labels & decorators will be hidden. Label are shown when hovering over the node
    - When zoomed to 30% scale, all labels, decorators, pod rings & icons will be hidden. Node shape remains the same, and background is either white, yellow or red. Background color is determined based on aggregate status of pods, alerts, builds and pipelines. Tooltip is available showing node name as well as the "things" which are attributing to the warning/error status.

**Checks required for approving Epic gherkin scripts PR:**

- [x] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
- [x] Add @to-do for automation possible scenarios
- [x] Add @regression or @smoke based on the importance of the scenario
- [x] Update the test scenarios count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)
- [x] Check for the linter issues by executing `yarn run gherkin-lint` on frontend folder [Skip epic number tags related linter issues]

**Fix**: https://issues.redhat.com/browse/ODC-6501

**Test Setup**:

Cluster should be running on local

Check the TAGS in frontend/packages/topology/integration-tests/cypress.json file be: ""@topology and @[odc-4944](https://issues.redhat.com//browse/odc-4944) and (@smoke or @regression or @pre-condition) and not (@manual or @to-do or @un-verified or @broken-test)"",

Command to execute:

Run a cluster locally and execute the command `yarn run dev` and then `npm run test-cypress-topology` simultaneously

Execute the topology-chart-area-visual.feature file

**Browser**
Chrome 99

**Test Execution Screenshot:**

### CODE REVIEW

File: frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature [hunk 1]
```diff
@@ -387,4 +399,32 @@ Feature: Topology chart area
...
+            Given user has installed Service Binding operator
+              And user has installed Redis Operator
+              And user is at developer perspective
+              And user is at Topology page chart view
+              And user has created a deployment workload named "node-j"
+              And user has created a operator backed service of "Redis" operator named "redis-standalone"
+              And user has created service binding connnector "test-connector1" between "node-j" and "redis-standalone"
+             When user clicks on service binding connector
+             Then user can see the Status section on service binding connnector sidebar
+              And user will see blue colored connector
```
Comment (jerolimov) [inline frontend/packages/topology/integration-tests/features/topology/topology-chart-area-visual.feature:416]:
```suggestion
             Then user can see the Status section on service binding connnector topology sidebar
              And user will see black colored connector
```

**Comment by sanketpathak** [comment]
The pr only affects our E2E tests
/label qe-approved
/label docs-approved
/label px-approved

**Pull request merged**